src/java/org/apache/cassandra/service/accord/AccordConfigurationService.java

```diff
@@ -213,6 +213,7 @@ public EpochDiskState truncateTopologyUntil(long epoch, EpochDiskState diskState
         }
     }
 
+    // TODO: should not be public
```

Comment:
```suggestion
    //TODO (???): should not be public
```

2 things

`//TODO` vs `// TODO`, not sure why intellij cares...

`(...)` should have a scope?